geom/src/main/java/org/geolatte/geom/codec/HANAWktVariant.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * This file is part of the GeoLatte project.
+ *
+ *     GeoLatte is free software: you can redistribute it and/or modify
+ *     it under the terms of the GNU Lesser General Public License as published by
+ *     the Free Software Foundation, either version 3 of the License, or
+ *     (at your option) any later version.
+ *
+ *     GeoLatte is distributed in the hope that it will be useful,
+ *     but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *     GNU Lesser General Public License for more details.
+ *
+ *     You should have received a copy of the GNU Lesser General Public License
+ *     along with GeoLatte.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright (C) 2010 - 2011 and Ownership of code is shared by:
+ * Qmino bvba - Romeinsestraat 18 - 3001 Heverlee  (http://www.qmino.com)
+ * Geovise bvba - Generaal Eisenhowerlei 9 - 2140 Antwerpen (http://www.geovise.com)
+ */
+
+package org.geolatte.geom.codec;
+
+import org.geolatte.geom.Geometry;
+import org.geolatte.geom.GeometryType;
+import org.geolatte.geom.crs.CoordinateReferenceSystem;
+
+import java.util.*;
+
+import static org.geolatte.geom.crs.CoordinateReferenceSystems.hasMeasureAxis;
+import static org.geolatte.geom.crs.CoordinateReferenceSystems.hasVerticalAxis;
+
+/**
+ * Punctuation and keywords for HANA EWKT/WKT representations.
+ *
+ * @author Jonathan Bregler, SAP
+ */
+class HANAWktVariant extends PostgisWktVariant {
+	
+	private final static List<HANAWktGeometryToken> GEOMETRIES = new ArrayList<HANAWktGeometryToken>();
+
+	private final static Set<WktKeywordToken> KEYWORDS;
+
+	protected HANAWktVariant() {
+	}
+
+	static {
+
+		// TODO -- this doesn't work well with LinearRings (geometry type doesn't match 1-1 to WKT types)
```

Comment:
Could you elaborate on this comment?

Reply:
I copied it from PostgisWktVariant.java. I figured if the comment is there it should also be here.